biz.aQute.tester.junit-platform/src/aQute/tester/junit/platform/plugin/ProjectTesterImpl.java

```diff
@@ -68,7 +73,9 @@ public boolean prepare() throws Exception {
 			Collection<String> testnames = getTests();
 			if (testnames.size() > 0) {
 				launcher.getRunProperties()
-					.put(TESTER_NAMES, Processor.join(testnames));
+					.put(TESTER_NAMES, Processor.join(testnames.stream()
```

Comment:
It seems busy to stream/map the list into a list and then join into a string. We can go straight there

```java
.put(TESTER_NAMES, testnames.stream()
  .map(ProjectTesterImpl::maybeQuote)
  .collect(Collectors.joining(",")));
```